specifyweb/frontend/js_src/lib/components/Toolbar/Query.tsx

```diff
@@ -170,18 +170,20 @@ function QueryList({
         {queries.map((query) => (
           <tr key={query.id} title={query.remarks ?? undefined}>
             <td>
-              <Link.Default
-                className="overflow-x-auto"
-                href={
-                  getQuerySelectUrl?.(query) ?? `/specify/query/${query.id}/`
-                }
-              >
-                <TableIcon
-                  label
-                  name={getModelById(query.contextTableId).name}
-                />
-                {query.name}
-              </Link.Default>
+              <>
```

Comment:
`<>` is not needed here

Reply:
Looks like redundant `<>` are still here. The other changes look good. As soon as this one is resolved we can merge